sentences/fr/_common.yaml

```
@@ -7,9 +7,51 @@ responses:
     no_device_class: "{{ area }} ne contient pas de {{ device_class }}"
     no_entity: "Aucun appareil ou entité appelé {{ entity }}"
     handle_error: "Une erreur est intervenue pendant le traitement"
-lists: {}
+lists:
+  color:
+    values:
+      - in: "blanche"
+        out: "white"
+      - in: "noire"
+        out: "black"
+      - in: "rouge"
+        out: "red"
+      - in: "orange"
+        out: "orange"
+      - in: "jaune"
+        out: "yellow"
+      - in: "verte"
+        out: "green"
+      - in: "bleue"
+        out: "blue"
+      - in: "violette"
+        out: "purple"
+      - in: "marron"
+        out: "brown"
+      - in: "rose"
+        out: "pink"
```

```suggestion
      - in: "blanc"
        out: "white"
      - in: "noir"
        out: "black"
      - in: "rouge"
        out: "red"
      - in: "orange"
        out: "orange"
      - in: "jaune"
        out: "yellow"
      - in: "vert"
        out: "green"
      - in: "bleu"
        out: "blue"
      - in: "violet"
        out: "purple"
      - in: "marron"
        out: "brown"
      - in: "rose"
        out: "pink"
```
For example : "Règle la lumière du bureau en bleu". 
Text to speech uses "bleu" and not "bleue".

---

I think you should add both versions, example in the case of "Adjust the light of the office with the blue colour" 
what do you think?